core/tests/puppeteer-acceptance-tests/user-utilities/exploration-editor-utils.ts

```diff
@@ -62,17 +77,87 @@ const explorationCompletionToastMessage = '.e2e-test-lesson-completion-message';
 
 export class ExplorationEditor extends BaseUser {
   /**
-   * Function to navigate to the creator dashboard page.
+   * Function to navigate to creator dashboard page
    */
-  async navigateToCreatorDashboard(): Promise<void> {
-    await this.page.goto(creatorDashboardUrl);
+  async navigateToCreatorDashboardPage(): Promise<void> {
+    await this.page.goto(creatorDashboardPage);
   }
 
   /**
-   * Function to click on the 'Create Exploration' button.
+   * Function to navigate to exploration editor
    */
-  async clickCreateExplorationButton(): Promise<void> {
+  async navigateToExplorationEditorPage(): Promise<void> {
     await this.clickOn(createExplorationButton);
+  }
+
+  /**
+   * Function to create an exploration with a content and interaction.
+   * This is a composite function that can be used when a straightforward, simple exploration setup is required.
+   *
+   * @param content - content of the exploration
+   * @param interaction - the interaction to be added to the exploration
+   */
+  async createExplorationWithContentAndInteraction(
+    content: string,
+    interaction: string
+  ): Promise<void> {
+    await this.updateCardContent(content);
+    await this.addInteraction(interaction);
+    await this.saveExplorationDraft();
+  }
+
+  /**
+   * Function to publish exploration
+   * @param {string} title - The title of the exploration.
+   * @param {string} goal - The goal of the exploration.
+   * @param {string} category - The category of the exploration.
+   */
+  async publishExploration(
+    title: string,
+    goal: string,
+    category: string
+  ): Promise<string | null> {
+    if (this.isViewportAtMobileWidth()) {
+      await this.page.waitForSelector(toastMessage, {
+        visible: true,
+      });
+      await this.page.waitForSelector(toastMessage, {
+        hidden: true,
+      });
+      await this.clickOn(mobileChangesDropdown);
+      await this.clickOn(mobilePublishButton);
+    } else {
+      await this.page.waitForSelector(
```

Comment:
Can we use the utility function waitForElementToBeClickable ?

Reply:
Here, I guess, It's fine without the existing one and the utility function as well, since the clickOn() already incorporates this waitForElementToClickable(). 
Updated accordingly. 
Thanks.